Index: LinearSearch.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/LinearSearch.py b/LinearSearch.py
new file mode 100644
--- /dev/null	(date 1628275230370)
+++ b/LinearSearch.py	(date 1628275230370)
@@ -0,0 +1,8 @@
+def linearSearch(arr, val): # time - O(N) ,space - O(1)
+    for i in range(len(arr)):
+
+        if arr[i] == val:
+            return i
+    return -7897
+print(linearSearch([34,76,45,23,59,7,89,0],23))
+print(linearSearch([34,76,45,23,59,7,89,0],69))
diff --git a/BinarySearch.py b/BinarySearch.py
new file mode 100644
